network.go

```diff
@@ -611,6 +617,10 @@ func (n *network) driver(load bool) (driverapi.Driver, error) {
 }
 
 func (n *network) Delete() error {
+	return n.delete(false)
+}
+
+func (n *network) delete(force bool) error {
```

Comment:
I thought about this some more and i think a network `force` delete must also NOT fail for `ActiveEndpointsError`.
If you think the case of `NetworkCleanup` should not behave that way, then please add the following check in `NetworkCleanup` before calling `delete(true)`.

```
 	numEps := n.getEpCnt().EndpointCnt()
 	if numEps != 0 {
 		continue
 	}
```